frontend/app/(dashboard)/bidding/[project_id]/submit/page.tsx

```
@@ -0,0 +1,242 @@
+  });
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    const { name, value } = e.target;
+    setFormData(prev => ({ ...prev, [name]: value }));
+  };
+
+  const validateForm = () => {
+    try {
+      bidSchema.parse({
+        cover_letter: formData.cover_letter,
+        bidding_amount: parseFloat(formData.bidding_amount) || 0 // Handle NaN case
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)